sentry-android-sqlite/src/main/java/io/sentry/android/sqlite/SentrySupportSQLiteOpenHelper.kt

```diff
@@ -0,0 +1,29 @@
+package io.sentry.android.sqlite
+
+import androidx.sqlite.db.SupportSQLiteDatabase
+import androidx.sqlite.db.SupportSQLiteOpenHelper
+
+class SentrySupportSQLiteOpenHelper(
+    private val delegate: SupportSQLiteOpenHelper
+) : SupportSQLiteOpenHelper by delegate {
+
+    private val sqLiteSpanManager = SQLiteSpanManager()
+
+    private val sentryDatabase: SupportSQLiteDatabase by lazy {
+        SentrySupportSQLiteDatabase(delegate.writableDatabase, sqLiteSpanManager)
+    }
+
+    override val writableDatabase: SupportSQLiteDatabase
```

Comment:
Actually, the `SupportSQLiteOpenHelper` itself says, for the `readableDatabase`:
```
Create and/or open a database.  This will be the same object returned by
[writableDatabase] unless some problem, such as a full disk,
requires the database to be opened read-only.  In that case, a read-only
database object will be returned.  If the problem is fixed, a future call
to [writableDatabase] may succeed, in which case the read-only
database object will be closed and the read/write object will be returned
in the future.
```
I still think it's an edge case, but we can easily create two wrappers, one for the `writeableDatabase` and one for the `readableDatabase`.

Reply:
ah ok, I see. In this case, I think it's fine to just start with the `writableDatabase` for now, we can also add the readable one later if necessary